src/components/Slider.js

```diff
@@ -1,137 +1,161 @@
 import React, { useMemo } from "react";
 import Slider from "react-slick";
-import { useLocation } from 'react-router-dom';
 
 const ArrowLeft = () => {
-  return (
-    <svg style={{
-      position: "absolute",
-      transform: "translate(-50%, -50%)"
-      }} xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-      <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={3} d="M15 19l-7-7 7-7" />
-    </svg>
-  )
-}
+	return (
+		<svg
```

Comment:
I think these arrows can be extracted in a shared folder, as they are used in other places. Please confirm @Katsu08

Reply:
Yes, is better create a component and pass as props the color or size